public/api/conf/2.0/schemas/wco/fileupload/FileUploadResponse.xsd

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns="hmrc:fileupload" xmlns:xs="http://www.w3.org/2001/XMLSchema"
```

Comment:
in FileUploadRequest i have targetNamespace="hmrc:fileupload2" so these are now inconsistent. I think they should be the same, and different from the current one (`hmrc:fileupload`) - what do you think?

Reply:
as per https://jira.tools.tax.service.gov.uk/browse/CDD-1777 
, `follow structure of the example .xml below` I thought it applied to field names as well. Let me talk to the BA